README.md

```diff
@@ -14,4 +14,10 @@ Cheri Allen has not been to a quiet beach with crystal blue water, white sand, a
 
 ### Citations:
 
+<<<<<<< HEAD
+Allen, Cheri. "I Want to Go Snorkeling Right Now." <i>Journal of Whiny North Americans</i>, 20 February 2025, pp. 1.
+
+Hello world
+=======
 Allen, Cheri. "I Want to Go Snorkeling Right Now." <i>Journal of Whiny North Americans</i>, 20 February 2025, pp. 12.
+>>>>>>> main
```

Comment:
Blocker: we need to remove the merge conflict git generated stuff. Lines 17, 21, 23.